the_mines/run.py

```diff
@@ -0,0 +1,6 @@
+from get_html import get_html
+from scrape_html import fussballdatenpunktde_matchday_results
+
+html_file = get_html()
+
+fussballdatenpunktde_matchday_results("data/" + html_file)
```

Comment:
I like that you've made a run file for this.  `get_data.py` may better describe what is happening here.